test/fixtures/browser/features/steps/browser_steps.rb

```diff
@@ -1,6 +1,35 @@
+def get_document_server_url
+  if Maze.config.aws_public_ip
+    "http://#{Maze.public_document_server_address}"
+  else
+    "http://#{get_private_hostname}:#{Maze.config.document_server_port}"
+  end
+end
+
+def get_maze_runner_url
+  if Maze.config.aws_public_ip
+      "http://#{Maze.public_address}"
+  else
+    "http://#{get_private_hostname}:#{Maze.config.port}"
+  end
+end
+
+def get_private_hostname
+  case Maze.config.farm
+  when :local
+    'localhost'
+  when :bs
+    'bs-local.com'
+  else # :bb
+    'local'
+  end
+end
```

Comment:
Is this right? What are `bs-local.com` and `local` and how do they translate into a reachable host?

We just use `Maze.config.bind_address` and `Maze.config.document_server_bind_address` in the SDK and it works as expected on BrowserStack, BitBar and locally